src/app/components/RuntimeEvents/RuntimeEventsDetailedList.tsx

```diff
@@ -16,11 +16,15 @@ export const RuntimeEventsDetailedList: FC<{
   isError: boolean
   pagination: false | TablePaginationProps
   showTxHash: boolean
-}> = ({ scope, events, isLoading, isError, pagination, showTxHash }) => {
+  filtered: boolean
+}> = ({ scope, events, isLoading, isError, pagination, showTxHash, filtered }) => {
   const { t } = useTranslation()
   return (
     <>
       {isError && <CardEmptyState label={t('event.cantLoadEvents')} />}
+      {!isError && filtered && (!pagination || pagination.selectedPage === 1) && !events?.length && (
+        <CardEmptyState label={t('tableSearch.noMatchingResults')} />
+      )}
       {isLoading && <TextSkeleton numberOfRows={10} />}
```

Comment:
https://pr-2078.oasis-explorer.pages.dev/mainnet/sapphire/address/0x289B7243138F376D8898F53Aedb63d943c382D02/events?event_type=accounts.burn
shows this while it is loading
<img width="1089" height="655" alt="image" src="https://github.com/user-attachments/assets/308167eb-bbbf-42f7-b4d7-04017628859a" />